matrix_sdk/src/models/room.rs

```diff
@@ -429,6 +430,19 @@ impl Room {
         }
     }
 
+    /// Receive a stripped state event for this room and update the room state.
+    ///
+    /// Returns true if the state of the `Room` has changed, false otherwise.
+    ///
+    /// # Arguments
+    ///
+    /// * `event` - The `AnyStrippedStateEvent` sent by the server for invited but not
+    /// joined rooms.
+    pub fn receive_stripped_state_event(&mut self, _event: &AnyStrippedStateEvent) -> bool {
+        // TODO do we want to do anything with the events from an invited room?
```

Comment:
Yes, probably setup a Room struct so we can check the display name of such a room as well.